### PR TITLE
use an unused parameter

### PR DESCRIPTION
**Pull request by miaoyq**
use an unused parameter

### CODE REVIEW

File: pkg/cmd/openshift/openshift.go [hunk 1]
```diff
@@ -117,7 +117,7 @@ func NewCommandOpenShift(name string) *cobra.Command {
 	root.AddCommand(cli.NewCommandCLI("cli", name+" cli", in, out, errout))
 	root.AddCommand(cli.NewCmdKubectl("kube", out))
 	root.AddCommand(newExperimentalCommand("ex", name+" ex"))
-	root.AddCommand(newCompletionCommand("completion", name+" completion"))
+	root.AddCommand(newCompletionCommand("completion SHELL", name+" completion"))
```
Comment (liggitt) [inline pkg/cmd/openshift/openshift.go]: don't include usage in the name

File: pkg/cmd/openshift/openshift.go [hunk 2]
```diff
@@ -203,7 +203,7 @@ func newCompletionCommand(name, fullName string) *cobra.Command {
 	out := os.Stdout
 
 	completion := &cobra.Command{
-		Use:     "completion SHELL",
+		Use:     name,
```
Comment (liggitt) [inline pkg/cmd/openshift/openshift.go]: `fmt.Sprintf("%s SHELL", name)`

**Comment by liggitt** [comment]
update docs with `hack/update-generated-docs.sh`

**Comment by miaoyq** [comment]
I have changed, PTAL, Thanks! @liggitt

**Comment by liggitt** [comment]
thanks, [merge]

**Pull request merged**